apps/web/README.md

```diff
@@ -1,34 +1,44 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-## Getting Started
+This app provides a platform intended to record reporting statistics involved with an Internationally Educated Nurse's hiring journey.  IEN is only resposible for Recruitment related milestones, all other data is submitted to the IEN portal through ATS\
 
-First, run the development server:
+> ** Authentication **\
+> See [Authentication](../../README.md#authentication)
 
-```bash
-npm run dev
-# or
-yarn dev
-```
+## Pages 
+***Each page requires particular roles and each role grants certain privileges, depending on roles, pages may look different***
```

Comment:
This line seems having two sentences. how about using conjunction or split them?